source/sonata_simulation.rst

```diff
@@ -492,3 +492,27 @@ example::
             "target": "Mosaic",
             "synapse_configure": "%s.Fac = 0 %s.Dep = 0"
   }
+
+meta_data
```

Comment:
nit: I usually consider `metadata` one word, so I think it's worth changing now.

Reply:
Sure, I've changed it.